src/main/java/frc/robot/utils/math/ArrayUtils.java

```diff
@@ -0,0 +1,52 @@
+package frc.robot.utils.math;
+
+public class ArrayUtils {
```

Comment:
Seems like `Arrays` support at least some of these operations.

Reply:
Would this be the equivalent of allMatch?
```
double[] comp = new double[measurement.value.length];
Arrays.fill(comp,-1);
Arrays.equals(measurement.value,comp);
```
Or is there a another method without streams?